### PR TITLE
3 september action cable

### PR DESCRIPTION
**Pull request by ShahirMansuri786**

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -23,5 +23,3 @@ def find_post
       @post = Post.find(params[:post_id])
```
Comment (aadi163) [inline app/controllers/likes_controller.rb]: Please correct indentation.

File: app/controllers/likes_controller.rb [hunk 2]
```diff
@@ -6,8 +6,8 @@ def new
 
   def create
     if user_signed_in?
-      @like = @post.likes.create(user_id: current_user.id)
-      redirect_to post_path(@post)
+      like = @post.likes.create(user_id: current_user.id)
+      redirect_to posts_path
     else
 	    redirect_to user_session_path
```
Comment (aadi163) [inline app/controllers/likes_controller.rb:12]: Correct indentation.  make sure you have to use only 2 spaces .

Comment (ShahirMansuri786) [reply app/controllers/likes_controller.rb:12]: Ok sure i will correct

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (ShahirMansuri786) [reply app/controllers/likes_controller.rb]: OK aditya sure

File: app/controllers/likes_controller.rb (hunk 2 above)
Comment (ShahirMansuri786) [reply app/controllers/likes_controller.rb:12]: Hello Aditya I have updated from my side but also it is showing incorrect in PR may be having some issues on system
It is showing correctly in my local code then I push the code

**Pull request merged**